src/main/kotlin/no/skatteetaten/aurora/gobo/ApplicationConfig.kt

```diff
@@ -125,19 +123,6 @@ class ApplicationConfig(
             })
             .clientConnector(clientConnector())
 
-    private fun exchangeStrategies(): ExchangeStrategies {
-        return ExchangeStrategies
-            .builder()
-            .codecs {
-                it.defaultCodecs().apply {
-                    maxInMemorySize(-1) // unlimited
```

Comment:
Vi bør vel sette spring.codec.max-in-memory-size i application.yaml

Reply:
ja, det glemte jeg. takk :)